src/app/commands/cmd_remove_layer.cpp

```diff
@@ -116,9 +116,16 @@ RemoveLayerCommand::RemoveLayerCommand()
 
 bool RemoveLayerCommand::onEnabled(Context* context)
 {
-  return context->checkFlags(ContextFlags::ActiveDocumentIsWritable |
-                             ContextFlags::HasActiveSprite |
-                             ContextFlags::HasActiveLayer);
+  if (!context->checkFlags(ContextFlags::ActiveDocumentIsWritable |
+                 ContextFlags::HasActiveLayer))
+    return false;
```

Comment:
You can keep the same flags here:

```suggestion
 if (!context->checkFlags(ContextFlags::ActiveDocumentIsWritable |
                          ContextFlags::HasActiveSprite |
                          ContextFlags::HasActiveLayer))
  return false;
```